src/hooks/useObjectButton.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  getStakingTransformFn,
+  useStakingWidgetContext,
+} from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget';
```

Comment:
Maybe we can shorten this? :D It's a long-ass import, sorry for my French

Reply:
This will be fixed by #354